schema-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaSchemaSerDe.java

```diff
@@ -180,8 +181,11 @@ protected void configureClientProperties(
       MapPropertyParser parser = new MapPropertyParser();
       metadata = parser.parse(config.getLatestWithMetadataSpec());
     }
-    ruleExecutors = initRuleObjects(config, RULE_EXECUTORS, RuleExecutor.class);
-    ruleActions = initRuleObjects(config, RULE_ACTIONS, RuleAction.class);
+    enableRuleServiceLoader = config.enableRuleServiceLoader();
```

Comment:
curious as to why we want to allow disabling service loader.

Reply:
Mainly for testing